app/views/partials/_properties.html.erb

```diff
@@ -5,6 +5,8 @@
       controller:  "<%= controller_name %>"
     },
     text: {
+      default_option: "<%= t('default_text.option') %>",
+      default_option_hint: "<%= t('default_text.option_hint') %>",
```

Comment:
I would suggest to change this lines to:

```
default_option: MetadataPresenter::DefaultText[:option],
default_option_hint: MetadataPresenter::DefaultText[:hint]
```

Although in order to work we need to add the option text on the gem: https://github.com/ministryofjustice/fb-metadata-presenter/blob/main/default_text/content.json

The reason for this is to have only one single source for default text content. Happy to do it myself if you like 🙏

Reply:
`MetadataPresenter::DefaultText[:option_hint]` just in case the hint text for a question or label ends up being different

Reply:
Changes made. Also required merge of https://github.com/ministryofjustice/fb-metadata-presenter/pull/106